workbench-core/base/src/constants/queryParams.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AttributeValue } from '@aws-sdk/client-dynamodb';
```

Comment:
this will affect the changes for filter and sort, I will set a reminder for myself to check this after merging our common code